qa/rpc-tests/coinbase_funding_streams.py

```diff
@@ -63,9 +65,11 @@ def run_test (self):
         self.nodes[0] = self.start_node_with(0, [
             "-mineraddress=%s" % miner_addr,
             "-minetolocalwallet=0",
-            fundingstream(0, 5, 9, [fs_addr, fs_addr, fs_addr]),
-            fundingstream(1, 5, 9, [fs_addr, fs_addr, fs_addr]),
-            fundingstream(2, 5, 9, [fs_addr, fs_addr, fs_addr]),
+            fundingstream(0, 5, 9, [fs_addr]),
+            fundingstream(1, 5, 9, [fs_addr]),
+            fundingstream(2, 5, 9, [fs_addr]),
```

Comment:
Not sure why all of these were changed; if we wanted to test "one addr" then only one needed to be changed.

Reply:
Only one address of the three being provided was being used, and TBH I think that being able to provide more addresses than is required is at best a potential source of confusion and bugs. We should change the verification logic to require exact correspondence, but I'm not sure whether we want to do that in this PR.